### PR TITLE
Typos in InstructorCourseEditPageUiTest, CommentAttributesTest and InstructorPrivilegesTest #6990

### LINKED ISSUE #6990: Fix typos in InstructorCourseEditPageUiTest and InstructorPrivilegesTest
Detail CheckStyle Report: https://htmlpreview.github.io/?https://github.com/xpdavid/CS2103R-Report/blob/master/codingStandard/spelling/test.html

InstructorCourseEditPageUiTest
Insturctor is not a word according to provided dictionary
``` java
        int unregisteredInsturctorIndex = 4;
```

~~CommentAttributesTest.java~~ Edit: removed in #7334
Memssage is not a word according to provided dictionary
``` java
List<String> errorMemssage = comment.getInvalidityInfo();
```

InstructorPrivilegesTest.java
Privilges is not a word according to provided dictionary
``` java
        Map<String, Boolean> sectionPrivilges = privileges.getSectionLevelPrivileges().get(sectionId);
```

### PR DESCRIPTION
**Pull request by beriwanravandi**
Fixes #6990 

This is my first attempt at contributing to an open source project! Simply just fixed typos in the aforementioned issue.

### CODE REVIEW

File: src/test/java/teammates/test/cases/datatransfer/InstructorPrivilegesTest.java [hunk 1]
```diff
@@ -248,28 +248,28 @@ public void testUpdatePrivilegeInSectionLevel() {
...
-        assertEquals(null, sectionPrivilges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION_COMMENT_IN_SECTIONS));
+        Map<String, Boolean> sectionPrivileges = privileges.getSectionLevelPrivileges().get(sectionId);
+        assertEquals(null, sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COURSE));
+        assertFalse(sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_STUDENT_IN_SECTIONS));
+        assertFalse(sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_COMMENT_IN_SECTIONS));
+        assertEquals(null, sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_GIVE_COMMENT_IN_SECTIONS));
+        assertEquals(null, sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COMMENT_IN_SECTIONS));
+        assertEquals(null, sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS));
+        assertEquals(null, sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_SUBMIT_SESSION_IN_SECTIONS));
+        assertEquals(null, sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION_COMMENT_IN_SECTIONS));
```
Comment (leeyimin) [inline src/test/java/teammates/test/cases/datatransfer/InstructorPrivilegesTest.java]: The line is too long, and needs to be broken up.
```
assertEquals(null,
        sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION_COMMENT_IN_SECTIONS));
```

File: src/test/java/teammates/test/cases/datatransfer/InstructorPrivilegesTest.java [hunk 2]
```diff
@@ -248,28 +248,28 @@ public void testUpdatePrivilegeInSectionLevel() {
...
-        assertEquals(null, sectionPrivilges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_SUBMIT_SESSION_IN_SECTIONS));
-        assertEquals(null, sectionPrivilges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION_COMMENT_IN_SECTIONS));
+        assertEquals(null, sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COURSE));
+        assertFalse(sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_STUDENT_IN_SECTIONS));
+        assertFalse(sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_COMMENT_IN_SECTIONS));
+        assertEquals(null, sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_GIVE_COMMENT_IN_SECTIONS));
+        assertEquals(null, sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COMMENT_IN_SECTIONS));
+        assertEquals(null, sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS));
+        assertEquals(null, sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_SUBMIT_SESSION_IN_SECTIONS));
+        assertEquals(null, sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION_COMMENT_IN_SECTIONS));
```
Comment (leeyimin) [inline src/test/java/teammates/test/cases/datatransfer/InstructorPrivilegesTest.java]: Same here.

**Comment by wkurniawan07** [comment]
Hi @beriwanravandi, any updates here?

**Comment by EdwardDunn** [comment]
Hi, i am looking for my first contribution. Is this issue needing any additional help?

**Comment by beriwanravandi** [comment]
I'll get this done today!

**Comment by beriwanravandi** [comment]
Hi @leeyimin, I'm getting an error saying "Project 'teammates' not found in root project 'teammates'" and not sure why this error is occurring. Any help would be great, thanks!

**Comment by whipermr5** [comment]
@beriwanravandi Did you rename your project folder outside of Eclipse or make any other changes to your project directory?

**Comment by whipermr5** [comment]
@beriwanravandi Closing due to inactivity. Feel free to reopen when you're ready to continue work on this :)

**Pull request closed without merging**